src/main/java/command/EditCommand.java

```diff
@@ -60,4 +60,10 @@ public void execute(EventList events, LocationList locations, BusStopList busSto
         ui.printEditEventMessage(newEvent);
         storage.writeFile(events.getEventList());
     }
+
+    public static String[] newEditInformation() {
```

Comment:
There is no difference made even you implement this static method. It's still not OOP.